spark/src/main/scala/geotrellis/spark/io/file/FileLayerProvider.scala

```diff
@@ -30,7 +30,10 @@ import java.io.File
 class FileLayerProvider extends AttributeStoreProvider
     with LayerReaderProvider with LayerWriterProvider with ValueReaderProvider with CollectionLayerReaderProvider {
 
-  def canProcess(uri: URI): Boolean = uri.getScheme.toLowerCase == "file"
+  def canProcess(uri: URI): Boolean = uri.getScheme match {
+    case str: String => if (str.toLowerCase == "file") true else false
+    case null => false
```

Comment:
This should be `true` I think? Something like `/Users/eugene/catalog` we can assume implies local file.

Reply:
I made that `false` so that its behavior is more in line with the other backends; but I can see why we'd want it to be `true`.